src/prpy/planning/ompl.py

```diff
@@ -192,38 +221,34 @@ def _SetPlannerRange(self, robot, ompl_args=None):
             # RRT-Connect's range to the same value used for collision
             # detection inside OpenRAVE.
             longest_extent = numpy.max(dof_ranges)
-            ompl_range = conservative_ratio * longest_extent - epsilon
+            ompl_range  = conservative_ratio * longest_extent - self.epsilon
+            ompl_range *= self.multiplier
```

Comment:
Also, I think the way this code in PrPy calculates the space resolution is inconsistent with the way it's calculated in `or_ompl` (see https://github.com/personalrobotics/or_ompl/blob/master/src/OMPLConversions.cpp#L154). Perhaps we changed the way it's calculated in `or_ompl` at some point without changing it here?